src/libaktualizr/utilities/utils.cc

```diff
@@ -731,6 +741,21 @@ class SafeTempRoot {
   boost::filesystem::path path;
 };
 
+#if defined(ANDROID)
+std::mutex Utils::storage_root_path_mutex_;
+std::string Utils::storage_root_path_;
+
+void Utils::setStorageRootPath(const std::string &storage_root_path) {
```

Comment:
Where/how is it used?

Reply:
I use it to point aktualizr to an allowed private storage on Android.